charts/graph-network-indexer/values.yaml

```diff
@@ -62,20 +63,148 @@ indexerDefaults:
     # -- (required) An endpoint to query the network subgraph
     network-subgraph-endpoint: null
 
-  env: {}
+  configService:
```

Comment:
I am a bit confused by the current structure. There was an indexerDefaults.config which is still in play. Now we also ahve a configService which is used to generate a configmap which itself uses a lot of the keys set here, but also plenty of the indexerDefaults.config. All the workloads seem to get this configmap now.

1) why do we have two, config and configService?
2) why is this one called configService, we also have an `indexer-service` component here but it is not specific to it
3) we can't override any of it per-component, which may be intended but then maybe we shouldn't call this "indexerDefaults" anymore

Reply:
@cjorge-graphops I've addressed all other comments. Regarding this one -it's a bit tricky the defaults that were there before under indexerDefaults.config were applicable to both indexer-agent and indexer-service and are still in use for indexer-agent. If I continue the list with tap-agent and indexer-rs only flags it will break indexer-agent. That's why i went this path. I could rename to something better but I was avoiding duplication of flags.